Sources/SmithyHTTPAuthAPI/SigningPropertyKeys.swift

```diff
@@ -1,13 +0,0 @@
-//
```

Comment:
This file was just moved to `SmithyHTTPAuthAPI/Context/` subdirectory, but git picked this up as deletion because in the new place, new keys were added from aws-sdk-swift as well.